engine/primitive/src/main/java/io/deephaven/engine/primitive/value/iterator/ValueIteratorOfChar.java

```diff
@@ -156,4 +140,45 @@ public long remaining() {
             }
         };
     }
+
+    /**
+     * Wraps a ValueIteratorOfChar with set number of prefix nulls, postfix nulls, or both. The result must be
+     * {@link #close() closed}.
+     *
+     * @param iterator The ValueIteratorOfChar to wrap
+     * @param prefixNulls The number of nulls to add to the beginning of the iterator
+     * @param postfixNulls The number of nulls to add to the end of the iterator
+     * @return A ValueIterator with the specified number of prefix and postfix nulls
+     */
+    static ValueIteratorOfChar wrapWithNulls(
```

Comment:
Coverage for `of`, `empty`, and `wrapWithNulls` is missing in `TestColumnIterators`.